charts/airflow/templates/_helpers/pods.tpl

```diff
@@ -142,6 +142,9 @@ EXAMPLE USAGE: {{ include "airflow.container.git_sync" (dict "Values" .Values "s
           name: {{ .Values.dags.gitSync.httpSecret }}
           key: {{ .Values.dags.gitSync.httpSecretPasswordKey }}
     {{- end }}
+    {{- if .Values.dags.gitSync.extraEnv }}
+    {{- toYaml .Values.dags.gitSync.extraEnv | nindent 6 }}
```

Comment:
Can you verify this works? I found I needed to change the nindent to `4`.

Reply:
oops, no, you're definitely correct -- this should be 4